frontend/graphql/getLocales.graphql

```diff
@@ -0,0 +1,5 @@
+query GetLocales {
+  pages(locale: "all") {
```

Comment:
I don't get what's the purpose of this.. Why filter by local equal to "all" ? First of all, doesn't `pages` already return all locales if it's not filtered? Second, where did you read that it's legit to pass "all" when you want all locales?

Reply:
It was an old query, I'll remove that.